src/screens/login/login.js

```diff
@@ -16,36 +19,32 @@ export default class LoginScreen {
     const context = { email: props.email, show: showProp };
     target.innerHTML = loginContent(context);
     target.querySelector('.login__content').addEventListener('submit', (event) => { this.logIn(event, target); });
+    return loginContent(context);
   }
 
-  logIn(event, target, router) {
+  logIn(event, target) {
     event.preventDefault();
     const formData = new FormData(event.target);
     const _email = formData.get('email');
     const _password = formData.get('password');
 
-    if (_email === 'u@u') {
-      this.router.navigate('main');
-      return;
-    }
-    if (_email === 'm@m') {
-      this.router.navigate('admin');
-      return;
-    } else {
-      this.render(target, { displayError: true, email: _email });
-    }
-    /*
-    fetch('/api/account/logon', {
+    fetch('http://localhost:3000/api/account/login', {
```

Comment:
Better move all api calls to separate file. For example 'login.service.js'
We can use different sources to get data (XHR, local storage etc.) So here we can left only `loginService.login(email, password).then(( ) => ....)`